### PR TITLE
Documentation Migration Preview

### PR DESCRIPTION
**Pull request by pmai**
Consolidated documentation migration preview updated against master. Will be kept up to date. Probably needs checks that recent v1.1.1 work is fully integrated.

### CODE REVIEW

**Review by max-rosin — APPROVED** [review]
From my perspective, everything looks fine.

**Comment by stefancyliax** [comment]
This is not up to date to branch 68_migrate_spec_topics_to_asciidoc. 
Currently it does not build due to "future_evolution.adoc" missing.

File: doc/osi-sensor-model-packaging_spec.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+include::./spec/sensor_data_outputs.adoc[leveloffset=+2]
+
+include::./spec/ground_truth_init_parameters.adoc[leveloffset=+2]
+
+include::./spec/traffic_command_inputs.adoc[leveloffset=+2]
+
+include::./spec/traffic_update_outputs.adoc[leveloffset=+2]
+
+include::./spec/future_evolution.adoc[leveloffset=+2]
```
Comment (stefancyliax) [inline doc/osi-sensor-model-packaging_spec.adoc:51]: This is not up to date to branch 68_migrate_spec_topics_to_asciidoc.
Currently it does not build due to "future_evolution.adoc" missing.

**Review by stefancyliax — CHANGES_REQUESTED** [review]

**Comment by stefancyliax** [comment]
CCB 07.07.2021:
Review will be done by @pmai

**Pull request closed without merging**